apps/frontend/src/utils/aave/AaveRepayTransactionsFactory.ts

```diff
@@ -92,13 +100,15 @@ export class AaveRepayTransactionsFactory {
 
   private async repayNative(
     amount: BigNumber,
+    isEntireDebt: boolean,
     borrowRateMode: BorrowRateMode,
     opts?: TransactionFactoryOptions,
   ): Promise<Transaction[]> {
     const nativeAsset = await getAssetDataByAddress(
       constants.AddressZero,
       BOB_CHAIN_ID,
     );
+    amount = isEntireDebt ? amount.add('1000000') : amount;
```

Comment:
Is it ok to have this value **'1000000'** hardcoded?

Reply:
Not ideal, maybe we can do a constant, but yeah, there's no specific rule for it other than "A bit more than debt". https://docs.aave.com/developers/periphery-contracts/wethgateway#repayeth for the contract field I can send uint256 but for value I don't think so... Any ideas are welcomed ))

Reply:
ok, I'll make note of this and maybe come back with smth in different pr